src/lib/processor/process_get_seat_manager_info.rs

```diff
@@ -0,0 +1,35 @@
+use ellipsis_client::EllipsisClient;
+use phoenix_seat_manager::{get_seat_manager_address, seat_manager::SeatManager};
+use solana_sdk::pubkey::Pubkey;
+
+use crate::helpers::market_helpers::get_seat_manager_data_with_market;
+
+pub async fn process_get_seat_manager_info(
+    client: &EllipsisClient,
+    market_pubkey: &Pubkey,
+) -> anyhow::Result<()> {
```

Comment:
Probably a good idea to check here that the market's authority is actually the Seat Manager